site/content/blog/2023-10-31-zap-2-14-0/index.md

```diff
@@ -0,0 +1,101 @@
+images:
+- https://www.zaproxy.org/blog/2023-10-31-zap-2.14.0/images/blog-2-14-0.png
+type: post
+tags:
+- blog
+- release
+date: "2023-10-31"
+authors:
+    - simon
+---
+ZAP 2.14.0 has just been [released](/docs/desktop/releases/2.14.0/), and adds support for Host Header Manipulation, ZAPit, API File Transfers, Graal JS Add-on Access, Postman collections, SBOMs, and more...
+
+![ZAPbot with 2.14.0 banner](images/blog-2-14-0.png)
+
+This release was made possible thanks to our Platinum Sponsor, the [Software Security Project](https://softwaresecurityproject.org/)
+
+![Software Security Project Logo](/img/supporters/ssp.png)
+
+### Rebranding and Docker Hub Move
+
+ZAP has had some minor rebranding changes as a result of the [move to the Software Security Project](/blog/2023-08-01-zap-is-joining-the-software-security-project/).
+
+As part of that move the official ZAP Docker images are being published to the 
+[Software Security Project Docker Hub Organisation](https://hub.docker.com/u/softwaresecurityproject). The OWASP links should continue to work for now but we recommend you change to use the new ones ASAP.
+
+Note that you can also pull the ZAP Docker images from [GitHub Container Registry](https://github.com/zaproxy/zaproxy/pkgs/container/zaproxy/versions?filters%5Bversion_type%5D=tagged).
+
+### Host Header Manipulation
+
+Host headers can now be manipulated in ZAP - we know many of you have been waiting for this for a long time!
+
+The [Break](https://www.zaproxy.org/docs/desktop/ui/tabs/break/), [Manual Request](https://www.zaproxy.org/docs/desktop/addons/requester/dialogs/) and [Requester](https://www.zaproxy.org/docs/desktop/addons/requester/tab/) dialogs all have a new "Update Host Header" button.
+This is enabled by default (to keep backwards compatibility) but if you turn this off then you will be able to specify your own host headers which will be sent to the target site.
+
+### ZAPit
+
+This release adds a new `-zapit` command line option to perform a quick ‘reconnaissance’ scan of the URL specified.
+
+For more details see the [ZAPit help page](/docs/desktop/addons/quick-start/zapit/)
+
+### API File Transfers
+
+You can now upload and download files to and from ZAP via the API.
+Note that this feature is disabled default as a security measure.
+
+For more details, including how to enable it, see the [API](/docs/desktop/start/features/api/#filexfer) help page.
+
+### Graal JS Add-on Access
+
+Since Oracle removed removed the Nashorn JavaScript engine from Java 15 anyone using Java 15+ has had to rely on the Graal JS add-on for Java Script support.
+Unfortunately due to classloader issues this was not able to access add-on classes, which significantly limited it's functionality.
+
+These issues have now been resolved which means that GraalJS is the recommended Java Script engine to use in ZAP.
```

Comment:
- `JavaScript`
- Mention that the existing scripts might need changes to work?
- `Graal JS` for consistency with the previous references.